go-app-ussd_nurse_rapidpro.js

```diff
@@ -199,35 +199,45 @@ go.app = function() {
             }).length !== 0;
         };
 
-       self.is_whatsapp_user = function(msisdn, wait_for_response) {
+        self.is_whatsapp_user = function(msisdn, wait_for_response) {
             var engage_config = self.im.config.services.engage || {};
             var api_url = engage_config.api_url;
-            var api_token = engage_config.api_token;
+            var api_user = engage_config.username;
+            var api_pass = engage_config.password;
 
-            var params = {
-                "contacts": [msisdn],
-                "blocking": wait_for_response ? "wait" : "no_wait"
-            };
-
-            return new JsonApi(self.im, {
-                headers: {
-                    'Authorization': ['Bearer ' + api_token]
+            return new HttpApi(self.im, {
```

Comment:
Why not use a JsonApi here too?

Reply:
Not sure why I ended up with the HttpApi, I thought it only added the header but I see now it decodes the result, then I don't have to do it manually. Thanks! 

Commit incoming.